gateway/src/shard/mod.rs

```diff
@@ -12,8 +12,41 @@
 //! Shards are configurable through the [`ShardBuilder`], which provides a clean
 //! interface for correctly configuring a shard.
 //!
-//! [`Event`]: ::twilight_model::gateway::event::Event
+//! # Member Chunking
+//!
+//! Requesting chunks of a guild's members may be done via [`Shard::command`]
+//! and [`RequestGuildMembers`]. For example, requesting chunks of members whose
+//! names start with "tw":
+//!
+//! ```no_run
+//! # #[tokio::main] async fn main() -> Result<(), Box<dyn std::error::Error>> {
+//! use std::env;
+//! use twilight_gateway::{shard::Shard, Intents};
+//! use twilight_model::{
+//!     gateway::payload::outgoing::RequestGuildMembers,
+//!     id::Id,
+//! };
+//!
+//! let intents = Intents::GUILD_PRESENCES;
```

Comment:
In this example presences are not requested therefore we do not need the intent but we need the guild members intent.

```suggestion
//! let intents = Intents::GUILD_MEMBERS;
```